frontend/src/Onboarding/LoginScreen.tsx

```diff
@@ -269,7 +268,7 @@ class LoginScreenComponent extends React.Component<Props, LoginScreenState> {
 
         <Subtitle>
           New here? Sign up{" "}
-          <Link style={{ color: "#EB5757" }} to="/signup">
+          <Link style={{ color: "#EB5757" }} to="/onboarding">
```

Comment:
Iman brings up a good point, I think we'll have to pass in a parameter to the Login screen to determine which route we redirect to. The issue is if a user continues as a guest, then from the home screen presses "Login", then they click the sign in link, they were never presented with the initial 4 onboarding questions in `/onboarding`.

Reply:
When the user attempts to sign up via the "login button" on the welcome page without going through onboarding, the serve responds with 422 error, since the payload is missing some information that is collected via onboarding page. When the user goes through "get started" they land on the signup page, so unless they toggle between sign up and login, they probably wont have to go through onboarding again

Reply:
why would we disable "continue as guest" from welcome page? do you mean via the login button?